ports/stm32f4/boards/feather_stm32f405_express/mpconfigboard.mk

```diff
@@ -14,6 +14,8 @@ MCU_VARIANT = stm32f4
 MCU_SUB_VARIANT = stm32f405xx
 MCU_PACKAGE = 64
 CMSIS_MCU = STM32F405xx
-LD_FILE = boards/STM32F405.ld
-TEXT0_ADDR = 0x08000000
-TEXT1_ADDR = 0x08010000
\ No newline at end of file
+
+LD_FS = boards/STM32F405_fs.ld # Default to internal FS
```

Comment:
Why internal FS? Doesn't the feather have an external SPI flash?

Reply:
It does. This just defines an internal filesystem anyway as a default, the way micropython does. Wastes 48K of flash, but if the user cuts the external flash, it'll allow the board to work. Do you want it cut, or a third option added?

Reply:
I would prefer to use the 48k for CircuitPython flash. We don't support removing external flash on any other board.